renderer/views/player.js

```diff
@@ -181,21 +181,22 @@ function renderLoadingSpinner (state) {
 function renderCastScreen (state) {
   var isChromecast = state.playing.location.startsWith('chromecast')
   var isAirplay = state.playing.location.startsWith('airplay')
+  var isDlna = state.playing.location.startsWith('dlna')
   var isStarting = state.playing.location.endsWith('-pending')
-  if (!isChromecast && !isAirplay) throw new Error('Unimplemented cast type')
+  if (!isChromecast && !isAirplay && !isDlna) throw new Error('Unimplemented cast type')
 
   // Show a nice title image, if possible
   var style = {
     backgroundImage: cssBackgroundImagePoster(state)
   }
 
-  // Show whether we're connected to Chromecast / Airplay
+  // Show whether we're connected to Chromecast / Airplay / DLNA
   var castStatus = isStarting ? 'Connecting...' : 'Connected'
   return hx`
     <div class='letterbox' style=${style}>
       <div class='cast-screen'>
         <i class='icon'>${isAirplay ? 'airplay' : 'cast'}</i>
```

Comment:
This means you can have two identical looking cast icons, one for Chromecast and one for Airplay, right?